### ISSUE TITLE
Implement multiple robot agents

### ISSUE THREAD

**Issue by why0504** [Shuijing725/CrowdNav_DSRNN#25]
hi,
I like your work very much,
I currently have an idea to do collaborative tasks among multiple robot agents.So I would like to ask you, how to implement multiple robot agents in the environment based on this github repository? Do you have any suggestions for this?

**Comment by Shuijing725** [comment]
See #22

**Comment by why0504** [comment]
Hi,
I've seen  [https://github.com/Shuijing725/CrowdNav_DSRNN/issues/22](url)
If I want to train two robots, then after adding the second robot instance, as follows:
 Under the two programs crowd_sim/envs/crowd_sim.py and crowd_sim/envs/crowd_sim_dict.py

`rob_RL = Robot(config, 'robot')
        rob2_RL = Robot(config,'robot2')
        self.set_robot(rob_RL)
        self.set_robot(rob2_RL)`
`ob['robot_node2'] = self.robot2.get_full_state_list_noV()`

I assume that both robots use the same reinforcement learning strategy, and the episode ends with both robots reaching the same target point. Then, referring to the training of the first robot, what procedures do I need to modify in the project file? Do I just need to modify the environment's code? For example, the related functions in crowd_sim_dict.py.
Secondly, does the render() function in crowd_sim.py also need to be modified?
Do I need to add code to train.py?
Sorry to bother you.
Thank you so much！

**Comment by Shuijing725** [comment]
Yes, besides the gym environment, multiple modifications are needed. For example, you probably need to modify the main scripts including `train.py` and `test.py`. 
Also, the code for network and ppo in [pytorchBaselines folder](https://github.com/Shuijing725/CrowdNav_DSRNN/tree/main/pytorchBaselines) needs to be adapted. For example, [dsrnn network](https://github.com/Shuijing725/CrowdNav_DSRNN/blob/main/pytorchBaselines/a2c_ppo_acktr/srnn_model.py) and [RL replay buffer](https://github.com/Shuijing725/CrowdNav_DSRNN/tree/main/pytorchBaselines/a2c_ppo_acktr) are only designed for single robot scenarios.

Besides changing our repo, another way to achieve your purpose is to look at open-source implementations of other multi-agent social navigation papers. For example, you can search for works that used environments like [this one](https://amrl.cs.utexas.edu/social_gym/).
